sqliterepo/synchro.c

```diff
@@ -259,8 +270,9 @@ zk_main_watch(zhandle_t *zh UNUSED, int type, int state, const char *path UNUSED
 				ss->zk_url);
 		return _reconnect(ss);
 	} else if (state == ZOO_CONNECTING_STATE) {
-		// FIXME(FVE): hardcoded value
-		if (grid_single_rrd_get_delta(ss->conn_attempts, now, 5) > 5) {
+		const guint64 delta = grid_single_rrd_get_delta(
+				ss->conn_attempts, now, disconnection_rrd_window);
```

Comment:
The creation of the `grid_single_rrd_s` should also be patched or `disconnection_rrd_window` may overflow.